match2/wikis/rocketleague/match_group_legacy_default.lua

```diff
@@ -52,6 +63,10 @@ function p._getMatchMapping(match, bracketData, bracketType)
 	end
 	round.G = round.G + 1
 
+	if string.match(bd.header or '', '^!l') then
```

Comment:
Could use a comment to explain the regex, to save some mental energy for the reader